test/rtp_decoder.c

```diff
@@ -4,14 +4,18 @@
  * decoder structures and functions for SRTP pcap decoder
  *
  * Example:
- * $ wget --no-check-certificate https://raw.githubusercontent.com/gteissier/srtp-decrypt/master/marseillaise-srtp.pcap
- * $ ./test/rtp_decoder -a -t 10 -e 128 -b aSBrbm93IGFsbCB5b3VyIGxpdHRsZSBzZWNyZXRz \
- *    < ~/marseillaise-srtp.pcap | text2pcap -t "%M:%S." -u 10000,10000 - - > ./marseillaise-rtp.pcap
+ * $ wget --no-check-certificate
```

Comment:
This comment isn't so readable anymore

Reply:
I'll see if I can go back and protect them from formatting. Not sure how well the clang-format off comments play inside comments but worth a shot!